include/gsl/gsl-lite.hpp

```diff
@@ -555,6 +577,48 @@ typedef integral_constant< bool, false > false_type;
 
 } // namespace std11
 
+// C++14 emulation:
+
+namespace std14 {
+
+#if gsl_HAVE( UNIQUE_PTR )
+# if gsl_HAVE( MAKE_UNIQUE )
+
+using std::make_unique;
+
+# elif gsl_HAVE( VARIADIC_TEMPLATE )
+
+template<typename T, typename... Args>
+std::unique_ptr<T> make_unique(Args&&... args)
+{
+    return std::unique_ptr<T>(new T(std::forward<Args>(args)...));
+}
+
+# else
```

Comment:
The following code (`unique_ptr<>` and rvalue references but no variadic templates) is exclusively for VS 2010/2012, right? I'd rather not support `make_unique<>()` there instead of starting to emulate variadic templates.

Reply:
Yes, it's only those compilers. I can take that branch out of the library, but since I use the functions for clarity in the unit tests, I need to add them to not_null.t.cpp. Is that okay?

Reply:
Sure.

Reply:
Will do, thanks.